docs/runbooks/mitx_residential_upgrade.md

```diff
@@ -0,0 +1,23 @@
+# Background
+MITx Residential upgrade typically takes place twice a year following OpenedX's named releases and happens between school semesters. To manage the upgrade, Devops has to perform a few taks before and during and scheduled maintancne window outlined below.
+
+## Several days prior to upgrade
+
+#### Communication
+- Once a date and time for the downtime is finalized, send request to post scheduled downtime to 3down. Here's an example of the message:
+    + The Residential MITx system (lms.mitx.mit.edu, studio.mitx.mit.edu and staging.mitx.mit.edu) will be unavailable starting at <time> for an estimated <time> hours while we perform routine upgrades. If you have any questions please contact mitx-support@mit.edu
```

Comment:
The `<time>` sections are being rendered literally which results in their being removed from the text. Consider either wrapping them in backticks (`) or using literal values (e.g. 9:00 a.m., 2 hours)